dts/bindings/ieee802154/atmel,rf2xx.yaml

```diff
@@ -38,3 +38,75 @@ properties:
       description: |
         Specifies the MAC address that was assigned to the network
         device
+
+    tx-pwr-map:
+      type: uint8-array
+      default: [ 0x00 ]
+      description: |
+        This is the Transmission Power Mapping array used to comply with local
+        regulations. By default this value set an output power above 0dBm for all
+        transceivers. This property must be used with tx-pwr-min and tx-pwr-max
+        for normal operations. The number of elements in the array is defined by
+        CONFIG_IEEE802154_RF2XX_TX_PWR_MAP_ENTRIES Kconfig property. The max
```

Comment:
This introduces a dependency from Kconfig into DT, which is a bit strange, no? Why is this necessary?

Reply:
My initial thought was to define this parameter as the basis for creating and validating the size of the table. As the drive allows the use of more than one radio, the table size is shared between them and it must be the same for both instances, as this parameter is in the device's configuration structure, I don't know if it's possible to get the size directly from the size array when there are two instances because it could have different values between the two devices.

Reply:
Can you do a BUILD_ASSERT that the table sizes are all the same within a DT_INST_FOREACH_STATUS_OKAY macro? Something like this:

```c
#if DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) > 1
#define THE_TABLE_SIZE(inst) ARRAY_SIZE(....(inst))
#define CHECK_TABLE_SIZE(inst) BUILD_ASSERT(THE_TABLE_SIZE(inst) == THE_TABLE_SIZE(0));
DT_INST_FOREACH_STATUS_OKAY(CHECK_TABLE_SIZE)
#endif
```